src/main/java/ox/app/game/ScoreBoard.java

```diff
@@ -0,0 +1,50 @@
+package ox.app.game;
+
+import ox.app.utility.RoundBuffer;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public class ScoreBoard {
+    private final Map<Player,Integer> playerPointsMap;
+    private final RoundBuffer playerBuffer;
+    private final Consumer<String> output;
```

Comment:
output is defined here and it is available for all of the functions, but only 2 of them need it.
Maybe better idea would be to pass it to those 2 methods instead of holding it in class?

Reply:
Changed :)